Makefile

```diff
@@ -104,50 +97,42 @@ requirements/lint.txt: requirements/tests.txt requirements/functests.txt
 # requirements/*.in files from disk ($(wildcard requirements/*.in)) and replace
 # the .in's with .txt's.
 .PHONY: requirements requirements/
+$(call help,make requirements,"compile the requirements files")
 requirements requirements/: $(foreach file,$(wildcard requirements/*.in),$(basename $(file)).txt)
 
-.PHONY: sure
-sure: checkformatting lint test coverage functests
+.PHONY: template
+$(call help,make template,"update from the latest cookiecutter template")
+template: python
+	@pyenv exec tox -e template -- $$(if [ -n "$${template+x}" ]; then echo "--template $$template"; fi) $$(if [ -n "$${checkout+x}" ]; then echo "--checkout $$checkout"; fi) $$(if [ -n "$${directory+x}" ]; then echo "--directory $$directory"; fi)
 
-.PHONY: update-pdfjs
-update-pdfjs: python
-	@tox -qe updatepdfjs
+DOCKER_TAG = dev
 
 .PHONY: docker
-docker: build
-	@git archive --format=tar HEAD > build.tar
-	@tar --update -f build.tar via/static
-	@gzip -c build.tar | docker build -t hypothesis/via:$(DOCKER_TAG) -
-	@rm build.tar
-
-.PHONY: web
-web: python
-	@tox -qe dev --run-command 'gunicorn -c conf/gunicorn/dev.conf.py --paste conf/development.ini'
-
-.PHONY: nginx
-nginx: python
-	@tox -qe dev --run-command 'docker-compose run --rm --service-ports nginx-proxy'
+$(call help,make docker,"make the app's docker image")
+docker:
+	@git archive --format=tar HEAD | docker build -t hypothesis/via:$(DOCKER_TAG) -
+
+.PHONY: docker-run
+$(call help,make docker-run,"run the app's docker image")
+docker-run:
+	@docker run \
+		--add-host host.docker.internal:host-gateway \
+		--env-file .docker.env \
+		--env-file .devdata.env \
+		-p 9082:9082 \
```

Comment:
I think some important stuff has been lost here. The main port has changed form 9083 to 9082, so if you go to it you get no content. There might be some other missing bits too.

Reply:
I think the port thing is the only change: it's the only difference I see and if I fix that one thing locally then it does work. But it's kind of a pain to fix properly: all our other projects use their Gunicorn port here in the Docker command except Via which uses a different port. I'll have to tweak the cookiecutter to allow for this. I wonder if it might be worth moving this whole command out into a `bin/make_docker_run` file that `Makefile` calls so that projects can take over this entire file for themselves if they want to

Reply:
^ Ok, that's merged and I've applied the change to this PR: `make docker-run` works now